example-server/app/public/form.js

```diff
@@ -172,6 +172,9 @@ app.controller('SetCustomerController', [
 
     scope.override = function () {
       scope.allowOverride = !scope.allowOverride;
+      if (!scope.allowOverride) {
+        scope.pixelPath = '';
+      }
```

Comment:
So I think the solution is just remove the if statement, since that probably makes it more clear what you're trying to do?